docker-entry.sh

```diff
@@ -2,13 +2,15 @@
 # Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
 # See License.txt for license information.
 
-echo "Starting MySQL"
-/entrypoint.sh mysqld &
+if [ "x${START_MYSQL,,}" != "xfalse" ]; then
```

Comment:
Sorry for the long delay here.
this is still relevant?
I dont see a use case for that. can you explain why we need that? thanks again